src/Oscoin/Consensus/Nakamoto.hs

```diff
@@ -37,7 +37,7 @@ import           Database.SQLite.Simple.ToField (ToField)
 
 -- | Target block time.
 blockTime :: Duration
-blockTime = 1 * seconds
+blockTime = 10 * minutes
```

Comment:
The target for the mainnet will likely be `1 * minutes` - that's what we've been modeling our supply curves from, with `60 osc` per block. This gives you a nice and intuitive `1 osc / second` emission rate.